README.md

```
@@ -113,71 +164,172 @@ curl -sSL https://install.python-poetry.org | python3 -
+### 5. Configure Environment
 
 ```bash
 cp env.example .env
 ```
 
-Edit `.env` with your MeetingBaas credentials.
+Edit `.env` with your MeetingBaas credentials and add the `BASE_URL` variable for production deployments.
+
+Example `.env` file:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)